### PR TITLE
readme updated and mockBarCrawlz added

### PR DESCRIPTION
**Pull request by filippo20u**

### CODE REVIEW

**Comment by LionSnarl** [comment]
Updated README for certain terminal commands.. Getting resource commands set up to start writing tests.

File: README.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ $ yarn add react-router-dom
 
 $ rails g resource Crawl title:string user_id:integer
 $ rails g resource Bar yelp_id:string name:string image-url:string rating:float adress1:string address2:string address3:string city:string zip_code:integer country:string state:string display_address:string phone:string display_phone:string
```
Comment (minachuong) [inline README.md]: `adress1` is misspelled

File: README.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ $ yarn add react-router-dom
...
-$ rails g resource JoinTable bar_id:integer crawl_id:integer
+$ rails g resource JoinTable bar_id:references crawl_id:references
```
Comment (minachuong) [inline README.md]: Looks like this is sorta notes for the team which is fine. Just know that usually README files are for displaying info about serving the project and documentation.

Comment (minachuong) [reply README.md]: I suggest using a different name for the `JoinTable` resource. Perhaps look at naming conventions for join tables.

**Review by minachuong — APPROVED** [review]

**Comment by LionSnarl** [comment]
Commit new README.md document to revise per Peer Review

**Pull request merged**